lib/challenge.js

```diff
@@ -48,7 +66,14 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name: "Spud",
+  species: "Dog",
+  breed: "Lab",
+  noise: = function () {
+    return "I'm a" + species + "I say Woof Woof.";
+}
```

Comment:
You never want to see two consecutive closing curly braces at the same level of indentation like this.This one should be indented one step, to show that it matches the opening curly brace from the function. The second one should have the same level of indentation as `var some_pet = {`, which is none here. This makes it easy to know which opening curly braces they match, and saves you reading time.